corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -194,3 +194,8 @@ def get_data_schema(request, domain, app_id=None, form_unique_id=None):
     if "pretty" in request.GET:
         kw["indent"] = 2
     return HttpResponse(json.dumps(data, **kw))
+
+
+@require_GET
```

Comment:
Performance was my main concern. However you raise a good point, I assumed that hitting any URL that goes through to django would extend the session timeout. Do you know if this is true or not?